.nuget/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="xunit.runner.console" version="2.0.0" />
```

Comment:
The console runner can generally be used to run tests on the console or on Travis CI, for example. However, due to that Mono bug, we can't use it right now. But the build script leverages that console runner to run all tests.